src/types/onyx/Transaction.ts

```diff
@@ -70,7 +88,7 @@ type Transaction = {
     modifiedWaypoints?: WaypointCollection;
     // Used during the creation flow before the transaction is saved to the server and helps dictate where the user is navigated to when pressing the back button on the confirmation step
     participantsAutoAssigned?: boolean;
-    pendingAction: OnyxCommon.PendingAction;
+    pendingAction: OnyxCommon.PendingAction | null;
```

Comment:
```suggestion
    pendingAction?: OnyxCommon.PendingAction;
```

Is possible to do this?

Reply:
yes!